### PR TITLE
Migrated Counter component. Added it to the StyleGuide.

### PR DESCRIPTION
**Pull request by FabioTranjan**
Migrated the **counter** component from MAT to be used in other Vavato Projects.

Preview:
Screen Shot 2021-08-10 at 16 39 07

### CODE REVIEW

File: src/ui/Counter.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+const CounterStyle = styled.input`
+  margin: 0;
+  width: 20px;
+  border: none;
+  color: black;
+  font-weight: 500;
+  font-size: 0.75rem;
+  text-align: center;
+  font-family: 'Merriweather Sans', -apple-system, BlinkMacSystemFont,
```
Comment (alexmreis) [inline src/ui/Counter.js:26]: Is this reqlly necessary? Fonts are set in the GlobalStyle

**Review by alexmreis — APPROVED** [review]

File: src/ui/Counter.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const CounterIconStyle = styled(FontAwesomeIcon)`
+  padding: 4px;
+  color: #c4c4c4;
+
+  :hover {
+    cursor: pointer;
+  }
+`
+
+function Counter({ counter, setCounter }) {
```
Comment (cedricpim) [inline src/ui/Counter.js:40]: We already have some components that store state (like `FoldablePanel` and `EllipsisMenu`), should we do the same here? Or it is a necessity to track the counter outside of the component?

**Review by cedricpim — COMMENTED** [review]
Like I mentioned in another MR, don't forget to export the components in `index.js`.

File: src/ui/Counter.js (hunk 1 above)
Comment (FabioTranjan) [reply src/ui/Counter.js:26]: Well noted

File: src/ui/Counter.js (hunk 2 above)
Comment (FabioTranjan) [reply src/ui/Counter.js:40]: Yes, the way we implemented it the counter is tracked outside the component, it can be a state or something else.

**Pull request merged**